department-of-veterans-affairs/va.gov-cms
# VACMS-6871: Enable content moderation for VHA facility non-clinical services

closes #6871 

- Configuration changes:
  - Added the "VAMC Facility Non-clinical Service" (vha_facility_nonclinical_service) to the editorial workflow.
  - Manually relocated the "revision_log" field to the new "Editorial workflow" fieldset, under the "moderation_state" field.

## Testing done


## Screenshots


## QA steps

As user editor
1. Go to `/node/add/vha_facility_nonclinical_service` to verify and confirm it matches other content types for the Editorial Workflow fieldgroup
1. Then validate Acceptance Criteria from issue
- [ ]  Content moderation has been enabled for vha_facility_nonclinical_service (at /admin/config/workflow/workflows/manage/editorial?destination=/admin/config/workflow/workflows)
- [ ]   /node/add/vha_facility_nonclinical_service matches other content types for the Editorial Workflow fieldgroup
    - [ ] Revision log message field appears in an editorial workflow fieldgroup, following patters on other content types

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
